src/scrapers/base-scraper-with-browser.js

```diff
@@ -94,7 +94,10 @@ class BaseScraperWithBrowser extends BaseScraper {
   async navigateTo(url, page) {
     const pageToUse = page || this.page;
     const response = await pageToUse.goto(url);
-    if (!response || response.status() !== OK_STATUS) {
+
+    const responseSuccess = response && response.status() === OK_STATUS;
+    const navigated = responseSuccess || pageToUse.url() === url;
```

Comment:
hi @baruchiro, reading [puppeteer goto documentation](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagegotourl-options]

>  NOTE page.goto either throws an error or returns a main resource response. The only exceptions are navigation to about:blank or navigation to the same URL with a different hash, which would succeed and return null.

I'm not sure this is the right condition. it might just work but from the documentation above is seems that the condition should be `response === null || response.status() === OK_STATUS`.  wdyt?

Reply:
You're right.
But now I think the only case where we'll have to throw an error is `response.status !== OK_STATUS`
Because in all other cases (`response === null || response.status() === OK_STATUS`, the error will be thrown by `goto`, or we got the expected result.